tox.ini

```diff
@@ -36,13 +36,13 @@ changedir =
     docs: docs
     flake8: .
     pkg: .
-    # pkg-nidcpower: bin/nidcpower
-    # pkg-nidmm: bin/nidmm
-    # pkg-nifgen: bin/nifgen
-    # pkg-niscope: bin/niscope
-    # pkg-niswitch: bin/niswitch
-    # pkg-nimodinst: bin/nimodinst
-    # pkg-nise: bin/nise
+    # pkg-nidcpower: generated/nidcpower
```

Comment:
don't know what this does but notice (orthogonal issue) `nidigital` is missing.

Reply:
They didn't do anything since they were commented out

All removed.